individual/jack/T012_TM1_3

```diff
@@ -0,0 +1,101 @@
+# Test 3: Dictionary Values: Individual Student Entries
+# Jack Roberts 101261505
+# Lab 4 ECOR1042
+# T012_TM1_3
+import T012_M1_load_data
+
+
+def T012_TM1_3() -> bool:
+    # --------------------------------------------------------------------------
+    types_failureskey_expected = (
+        str, int, float, int, int, float, float, float)
+    dic = (T012_M1_load_data.student_failures_dictionary(
+        'student-mat.csv'))
+    pass_count_failures = 0
+    error_count_failures = 0
+    for key in dic:
+        for student in dic[key]:
+            data_types = []
+            for key in student:
+                data_types.append(type(student[key]))
+            for i in range(len(data_types)):
+                # checks if the elements of data_types match the values you hard coded in types_failureskey_expected below
+                if data_types[i] == types_failureskey_expected[i]:
+                    pass_count_failures += 1
+                else:
+                    print(str(data_types[i]) + "=="
+                          + str(types_failureskey_expected[i]) + "  ERROR failures")
+                    error_count_failures += 1
+
```

Comment:
`            break
        break`
the "for key in dic" and "for student in dic[key]" actually only need to go through their first iteration.. they don't need to continue.. once the "for key in student" and "for i in range(len(data_types))" execute.. you're done for that dictionary.